Eastern-Research-Group/mywaterway
# Feature/hmw 73 add counts to accordion

## Related Issues:
* https://jira.epa.gov/browse/HMW-73

## Main Changes:
* Fixed the counts on the Swimming, Eating Fish, Aquatic Life, and Drinking Water tabs.
  * This was caused by c3e2ff6. I ended up reverting part of that commit, since the `field`  variable comes from the web service.
* Added counts to the accordion headers to make the app more consistent.
* Added counts to the sub accordion content on the Protect tab.

## Steps To Test:
1. Navigate to http://localhost:3000/community/040302020702/protect
2. Expand each accordion item, except for the "Watershed Health Scores" item
3. Verify each one has a count, except for the "Watrershed Health Scores"
4. Go through each main tab (i.e. Overview, Swimming, etc.)
5. Verify the accordion headers have counts in them now.
